### PR TITLE
Minor fixes to KMT invoke tasks

### PR DESCRIPTION
**Pull request by usamasaqib**
### What does this PR do?
This PR makes some changes to the invoke tasks for interacting with the kernel matrix testing system.

- It makes the `--branch` parameter implicit.
- Fixes to the destroy task.
- Adds some missing dependencies in the setup script.

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature. 
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

**Review by val06 — APPROVED** [review]

File: tasks/kernel_matrix_testing/stacks.py [hunk 1]
```diff
@@ -183,6 +184,10 @@ def destroy_ec2_instances(ctx, stack):
 
         ips.append(o.split(' ')[1])
 
+    if len(ips) == 0:
+        info(f"[+] No ec2 instance to terminate in stack")
```
Comment (chouetz) [inline tasks/kernel_matrix_testing/stacks.py:188]: useless f-string

**Review by chouetz — APPROVED** [review]
ok

**Comment by KevinFairise2** [comment]
Should we really be owner of the `kmt.py` file? Maybe you could add it to your team in the CODEOWNER file?

**Review by L3n41c — APPROVED** [review]
LGTM for the `Agent e2e testing`-owned files.

**Pull request merged**